tests/spicy/tools/spicy-driver-file.spicy

```diff
@@ -0,0 +1,17 @@
+# @TEST-EXEC: spicy-driver -f input.dat %INPUT >>output
+# @TEST-EXEC: spicy-build %INPUT
+# @TEST-EXEC: ./a.out -f input.dat >>output
+# @TEST-EXEC: btest-diff output
+
+module Test;
+
+public type Foo = unit {
+    x: bytes &eod;
+    on %done { print self; }
+};
+
+@TEST-START-FILE input.dat
```

Comment:
```suggestion
# @TEST-START-FILE input.dat
```

Reply:
btest will take out the line, so won't make a difference. Just looked at existing tests, and we are inconsistent in whether we're using  '#" with `@TEST-START-FILE` or not ...

Reply:
Putting it in a comment would make it easier to distinguish non-Spicy meta data from actual code e.g., for syntax highlighting. But OTOH, this file isn't valid Spicy code anyway due to `@TEST-START-FILE` etc.

Reply:
yeah, right. I'll put it in, and we should unify it sometime